### DISCUSSION TITLE
How to import unocss in @quasar/app-vite project import?

### DISCUSSION THREAD

**Discussion by lizhongyue248** [quasarframework/quasar#12873] [General - Components / Directives / etc] [answered]
I can not find `matn.ts` file. So, I don't know which file to put `import 'uno.css'`.

I tried to `quasar.config.js`
```
    boot: [
      'i18n',
      'axios',
      '~unocss/uno.css'
    ],
```
But I got a error.

What should I do?

**Comment by rowild** [comment]
Would like to know that, too. @lizhongyue248 Did you find a solution already?

I tried to add unocss to the vitePlugins section of quasar.config.js:

```js
module.exports = configure(function (ctx) {
  return {
    build: {
      vitePlugins: [
        ['@unocss/vite', {
          attributify: true, // enabled @unocss/preset-attributify,
          icons: true, // enabled @unocss/preset-icons
          uno: true, // enabled @unocss/preset-uno
          wind: true, // enabled `@unocss/preset-wind
        }]
      ]
    }
  }
}
```

And in my `layout/MainLayout.vue` I added this line, which is taken from a complaint in the terminal:

```js
// MainLayout.vue

<script setup>
import 'uno.css'

...
</script>
```

However, that would only import a uno.css file, which I do not have, so no idea, why this warning is shown... I removed that libe again.

**Comment by rstoenescu** [comment, marked as answer]
If you want to include css into your app, then use quasar.config.js > `css` instead of `boot` (as boot files are script files, not css/sass/etc)

**Reply by rowild** [reply]
@rstoenescu How does this work for unocss? Can you provide an example configuration, please?

**Reply by MilosPaunovic** [reply]
[https://quasar.dev/quasar-cli-webpack/quasar-config-js#property-css](https://quasar.dev/quasar-cli-webpack/quasar-config-js#property-css)

**Reply by rowild** [reply]
@MilosPaunovic Thank you for your feedback! I have the feeling you don't know what unocss is – sorry if I am wrong. Anyway, it is not a css library, but a css engine (like Tailwind's JIT compiler). The link does not explain anything with regards to unocss. Further features of unocss, like how to implement attributify and presets, are not even remotely covered.

If I'm wrong, please post a working example. Thanks!

**Reply by MilosPaunovic** [reply]
You are completely right, I got ahead of myself.
Ignore it and let's wait for someone else from the community with a better answer. 🙂

**Reply by rowild** [reply]
The impenetrable thicket of frameworks - impossible to know them all! 
Thank you for taking part and trying to help! Your effort is much appreciated! :-)

**Reply by rowild** [reply]
@rstoenescu Please remove your "marked as answer", since it does not answer the problem.

**Comment by rowild** [comment]
@lizhongyue248 @rstoenescu Please look here: https://github.com/unocss/unocss/issues/790
